README.md

```diff
@@ -137,6 +137,10 @@ This plugin assumes that the `mmdc` mermaid commandline tool is available, and w
 
 If there are errors with the mermaid syntax, it is treated as a normal codeblock and the warning is logged.
 
+### Linked Header Blocks
+
+Replace headers with links back to the originating notion block.
```

Comment:
This makes it sound like the headers are no longer headers as they're replaced by links. Perhaps say "Make headers have links back to the originating notion block."